_posts/blog/2018-10-31/2018-10-31-why-we-are-integrating-ethereum-into-openbazaar.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: "Why We Are Integrating <span class='post-title-extra'>Ethereum</span> into OpenBazaar"
+layout: post
+date: '2018-10-31 08:30:00 -0600'
+social_title: 'Why We Are Integrating Ethereum into OpenBazaar'
+social_description: 'Why We Are Integrating Ethereum into OpenBazaar'
+---
+
+In a previous article we explained [how we’re integrating Ethereum into OpenBazaar](https://openbazaar.org/blog/Escrow-Smart-Contract-Specification-in-OpenBazaar/). In this article we want to talk about _why_ we're integrating [Ethereum](https://www.ethereum.org/).
+
+#### Bitcoin Beginnings
+
+
+Then, in late 2017, bitcoin transaction fees began to rise. [A lot](https://bitinfocharts.com/comparison/bitcoin-transactionfees.htm). Using bitcoin for payments on OpenBazaar became impractical for months. Since payments in non-bitcoin currencies weren't native on OpenBazaar (they were simply Shapefshifted into bitcoin payments) that meant _all_ payments on OpenBazaar were impractical. Users wanted to use alternative currencies, but we were -- effectively -- forcing them to use bitcoin. Use of the platform ground to a halt. It was obvious something needed to change.
+
+#### Becoming Crypto-Inclusive
+
+We want users to be able to trade using **any cryptocurrency they desire**, without exposure to risks caused by other chains. In early 2018, in response to urgent demand from our users, we began integrating native altcoin payments into OpenBazaar. We're refactoring our codebase to make it much easier to integrate new cryptocurrencies natively. The low hanging fruit are coins that are forks of Bitcoin, as their codebases are closest to what OpenBazaar was already supporting. [Bitcoin Cash](https://www.bitcoincash.org/) and [Zcash](https://z.cash/) were the first such integrations, and [Litecoin](https://litecoin.org/) integration is almost complete.
+
+#### Enter Ethereum
+
+A natural continuation of that effort is to integrate Ethereum. Since Ethereum doesn't share Bitcoin's codebase, it will be more challenging to integrate than forks of Bitcoin. But we think it's worth the effort because:
```

Comment:
Should "will be more challenging" be "has been more challenging" or "is more challenging"? We're in the middle of doing it now.